#this message contains each satellites's position, receiver's psudorange, receiver's position
Header header
gnssToU[] meas
float64 latitude
float64 longitude
float64 altitude
